### PR TITLE
try cross compile with Scala 3.2.0 No need review.

### PR DESCRIPTION
**Pull request by He-Pin**
ontop of https://github.com/akka/akka/pull/31578
try to cross compile with scala 3.2.0

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -45,8 +45,10 @@ object Dependencies {
   val scala213Version = "2.13.8"
   // To get the fix for https://github.com/lampepfl/dotty/issues/13106
   // and restored static forwarders
-  val scala3Version = "3.1.2"
-  val allScalaVersions = Seq(scala213Version, scala212Version, scala3Version)
+  val scala31Version = "3.1.3"
+  val scala32Version = "3.2.0"
```
Comment (patriknw) [inline project/Dependencies.scala:49]: Thanks for trying this @He-Pin 

What I have heard from @SethTisue is that we can build with Scala 3.1.x and that will work fine for libraries/applications that are using 3.2.x. Scala 3 minor versions are backwards compatible.

Therefore I suggest that we don't cross build for Scala 3.2.

Comment (He-Pin) [reply project/Dependencies.scala:49]: I try to use `+ 3.x` and `++ 3.x` both not work now .
https://github.com/sbt/sbt/issues/7018

Comment (He-Pin) [reply project/Dependencies.scala:49]: @patriknw I try this to address find the issue of dotty 3.2 https://github.com/lampepfl/dotty/issues/16018.
And yes, compiles with 3.1 can consumed by 3.2.

Comment (SethTisue) [reply project/Dependencies.scala:49]: agree, it's only necessary to build for one Scala 3 version

**Comment by patriknw** [comment]
Can we close this now?

**Pull request closed without merging**